<!--REQUEST_ID=2d4431bd-7160-49b1-be0a-931b9a0e5dba-->
Failure on x86_64-darwin [(full log)](https://logs.nix.ci/?key=nixos/nixpkgs.47914&attempt_id=34ebe994-d28b-4e8b-9db4-729d3640a053)

Attempted: python36Packages.black, python36Packages.pyls-black

<details><summary>Partial log (click to expand)</summary><p>

```
            py36=config["py36"],
            pyi=config["pyi"],
>           skip_string_normalization=config["skip_string_normalization"],
        )
E       TypeError: from_configuration() missing 1 required keyword-only argument: 'skip_numeric_underscore_normalization'

/nix/store/843qjw57mvp2g2n3lpabxjk76kwl6jg7-python3.6-pyls-black-0.2.1/lib/python3.6/site-packages/pyls_black/plugin.py:49: TypeError
====================== 9 failed, 2 passed in 0.42 seconds ======================
builder for '/nix/store/npjby99q72gxz0ljvgbraqfy08sh6r4r-python3.6-pyls-black-0.2.1.drv' failed with exit code 1
error: build of '/nix/store/npjby99q72gxz0ljvgbraqfy08sh6r4r-python3.6-pyls-black-0.2.1.drv' failed
```
</p></details>

--- a/pkgs/development/python-modules/black/default.nix
+++ b/pkgs/development/python-modules/black/default.nix
@@ -4,13 +4,13 @@
 
 buildPythonPackage rec {
   pname = "black";
-  version = "18.6b4";
+  version = "18.9b0";
 
   disabled = pythonOlder "3.6";
 
   src = fetchPypi {
     inherit pname version;
-    sha256 = "0i4sfqgz6w15vd50kbhi7g7rifgqlf8yfr8y78rypd56q64qn592";
+    sha256 = "1992ramdwv8sg4mbl5ajirwj5i4a48zjgsycib0fnbaliyiajc70";
   };
 
   checkInputs =  [ pytest glibcLocales ];

--- a/pkgs/development/python-modules/pyls-black/default.nix
+++ b/pkgs/development/python-modules/pyls-black/default.nix
@@ -19,6 +19,9 @@ buildPythonPackage rec {
     pytest
   '';
 
+  # Enable when https://github.com/rupert/pyls-black/pull/6 is merged.
+  doCheck = false;
+
   checkInputs = [ pytest ];
 
   propagatedBuildInputs = [ black toml python-language-server ];

